playbooks/ansible-test-base/run.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
make sure to delete the original file, as we are moving it

Reply:
I will remove the original file, once this runs successfully with openvswitch. The 'ansible-test-integration-base' is being used by other jobs.